balena/models/device_os.py

```diff
@@ -68,14 +68,21 @@ def get_config(self, app_id, options):
             endpoint=self.settings.get('api_endpoint')
         )
 
-    def download(self, raw=None, **data):
+    def download(self, raw=False, is_dev_image=False, **data):
         """
         Download an OS image. This function only works if you log in using credentials or Auth Token.
 
         Args:
             raw (bool): determining function return value.
+            is_dev_image (Optional[bool]): select development image instead of production image. defaults to False.
             **data: os parameters keyword arguments.
-                Details about os parameters can be found in parse_params function
+                version (str): the Balena OS version of the image. The SDK will try to parse version into semver-compatible version, unsupported (unpublished) version will result in rejection.
```

Comment:
I think this should be `balenaOS`, as per our site
See: https://www.balena.io/os/

Also how about:
```suggestion
                version (str): the balenaOS version of the image. The SDK will try to parse the provided version as a semver-compatible version. Providing unavailable versions will result a rejection.
```

Reply:
updated!